elements/lisk-p2p/src/peer/base.ts

```diff
@@ -157,6 +159,15 @@ export class Peer extends EventEmitter {
 				return;
 			}
 
+			// Protocol RCP request limiter LIP-0004
+			if (PROTOCOL_REMOTE_EVENT_RCP_EVENTS.has(rawRequest?.procedure)) {
+				if (this._protocolRCPCounter <= PROTOCOL_REMOTE_EVENT_RCP_EVENTS.size) {
+					this._protocolRCPCounter += 1;
+				} else {
+					return;
+				}
```

Comment:
I would write the above code with guard conditions. 

```
if (this._protocolRCPCounter > PROTOCOL_REMOTE_EVENT_RCP_EVENTS.size) {
    return; 
}

this._protocolRCPCounter += 1;
```
It increase readability.